### PR TITLE
Bump v1.0.12

### PR DESCRIPTION
**Pull request by ajzkk**
### Description
- Modify library version

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -13,6 +13,9 @@ import (
 	"github.com/omise/omise-go/internal"
 )
 
+// LibraryVersion to include in user agent
+var LibraryVersion = "1.0.12"
```
Comment (danfowler) [inline client.go:17]: @ajzkk I think you can just make this `libraryVersion` so it won't be exported.

Comment (ajzkk) [reply client.go]: Updated

File: client.go [hunk 2]
```diff
@@ -13,6 +13,9 @@ import (
 	"github.com/omise/omise-go/internal"
 )
 
+// LibraryVersion to include in user agent
```
Comment (danfowler) [inline client.go:16]:
```suggestion
// Library version to include in user agent
```

File: client.go [hunk 3]
```diff
@@ -113,7 +116,7 @@ func (c *Client) buildJSONRequest(operation internal.Operation) (*http.Request,
 }
 
 func (c *Client) setRequestHeaders(req *http.Request, desc *internal.Description) error {
-	ua := "OmiseGo/2015-11-06"
+	ua := "OmiseGo/" + libraryVersion
```
Comment (aashishgurung) [inline client.go:119]: Can we add API version as well `OmiseAPI/2015-11-06`?

Comment (ajzkk) [reply client.go:119]: It is already there `req.Header.Add("Omise-Version", c.APIVersion)`

**Review by danfowler — APPROVED** [review]

**Review by aashishgurung — APPROVED** [review]

**Pull request merged**